restapi/opts.go

```diff
@@ -34,22 +40,12 @@ func Auth(auth Authorizer) Option {
 // X509 setup trust certificate
 func X509(cert *x509.Certificate) Option {
```

Comment:
Not really related to this MR, but IMO X509  is too generic term here. TrustAnchor() - as in the comment above - would be more descriptive.

Reply:
X509 is an inheritance from origin. `TrustAnchor` just works.